### PR TITLE
remove some redundant variables

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

**Review by kblok — APPROVED** [review]

File: src/PlaywrightSharp.Tests/ElementHandleScreenshotTests.cs [hunk 1]
```diff
@@ -417,7 +417,7 @@ public async Task ShouldTakeElementScreenshotWhenDefaultViewportIsNullAndRestore
                 <div class=""to-screenshot""></div>");
 
             var sizeBefore = await page.EvaluateAsync<ViewportSize>("() => ({ width: document.body.offsetWidth, height: document.body.offsetHeight })");
-            var elementHandle = await page.QuerySelectorAsync("div.to-screenshot");
+            await page.QuerySelectorAsync("div.to-screenshot");
             byte[] screenshot = await page.ScreenshotAsync();
```
Comment (kblok) [inline src/PlaywrightSharp.Tests/ElementHandleScreenshotTests.cs:421]: We should have called `elementHandle.ScreenshotAsync();` here.

File: src/PlaywrightSharp.Tests/ElementHandleConvenienceTests.cs [hunk 2]
```diff
@@ -73,7 +73,7 @@ public async Task InnerTextShouldThrow()
             Assert.Contains("Not an HTMLElement", exception1.Message);
 
             var handle = await Page.QuerySelectorAsync("svg");
-            var exception2 = await Assert.ThrowsAnyAsync<PlaywrightSharpException>(() => handle.GetInnerTextAsync());
+            await Assert.ThrowsAnyAsync<PlaywrightSharpException>(() => handle.GetInnerTextAsync());
             Assert.Contains("Not an HTMLElement", exception1.Message);
```
Comment (kblok) [inline src/PlaywrightSharp.Tests/ElementHandleConvenienceTests.cs:77]: We should have used `exception2` here.

Comment (SimonCropp) [reply src/PlaywrightSharp.Tests/ElementHandleConvenienceTests.cs]: done

File: src/PlaywrightSharp.Tests/ElementHandleScreenshotTests.cs (hunk 1 above)
Comment (SimonCropp) [reply src/PlaywrightSharp.Tests/ElementHandleScreenshotTests.cs]: done

**Comment by kblok** [comment]
It seems we broke ShouldTakeElementScreenshotWhenDefaultViewportIsNullAndRestoreBack

**Comment by SimonCropp** [comment]
so is the test correct? and we have exposed a bug?

**Review by avodovnik — CHANGES_REQUESTED** [review]
@SimonCropp we've made a ton of changes since you submitted this PR. If you're up for it, could you rebase and resubmit?

**Comment by avodovnik** [comment]
I've gone ahead and rebased, updated and created a new PR under #1466

**Pull request closed without merging**